themes/dark.json

```diff
@@ -570,11 +569,11 @@
 		},
 		{
 			"scope": [
-				"keyword.bracket-command.stellaris.yml",
-				"keyword.loc-key.stellaris.yml"
+				"bracket-command.stellaris.yml",
+				"loc-key.stellaris.yml"
```

Comment:
What is the reason behind this change?

Reply:
With keyword.loc-key.stellaris, you'll notice that things like $loc-key$ were always the same color as "keyword," even when when it was nested between §H and §!, for example. This is because it was picking up the color from the "keyword" scope instead of the color of the scope "keyword.color-code-h.stellaris." I found that this change fixed that.